pkg/machines/libvirt-dbus.js

```diff
@@ -1113,9 +1138,30 @@ function startEventMonitorNetworks(connectionName, dispatch) {
                 dispatch(getNetwork({ connectionName, id:objPath }));
                 break;
             case Enum.VIR_NETWORK_EVENT_STOPPED:
-                dispatch(getNetwork({ connectionName, id:objPath, updateOnly: true }));
+                dispatch(getNetwork({ connectionName, id:objPath }))
+                        .catch(ex => {
+                            if (!ex.message.includes("Network not found"))
+                                return;
+
+                            // In case the resource was transient the getX api above failed to fetch resource information
+                            // since it just got undefined.
+                            // Instead manually update the resource's active state and continue with removal.
+                            dispatch(updateOrAddNetwork({
+                                connectionName,
+                                id: objPath,
+                                active: false,
+                            }, true));
+                            // transient VMs don't have a separate Undefined event, so remove them on stop
+                            dispatch(removeNetwork({ connectionName, id: objPath, transientOnly: true }));
```

Comment:
Since you pass transientOnly parameter, shouldn't that parameter be added to removeNetwork function definition in actions/store-actions.js?

Reply:
Yes, good catch.